src/main/java/com/github/onsdigital/babbage/configuration/Handlebars.java

```diff
@@ -67,6 +71,10 @@ public boolean isReloadTemplateChanges() {
         return reloadTemplateChanges;
     }
 
+    public boolean getEnableLoop11() {
```

Comment:
Slightly pedantic but Java naming convention for boolean accessor methods is to use `is` or `can` instead of get. Something like ` isLoop11Enabled()` reads better from the caller POV.